docs/en/1-Experiments/Selecting-an-Image.md

```diff
@@ -21,11 +21,20 @@ When selecting an image, you have 3 main options:
     data transformation, numerical simulation, statistical
     modelling, machine learning and more.
 
-The jupyter notebooks are great launchpads for analytics including machine learning. The `jupyterlab-cpu` image gives a good core experience for python, including common packages such as `numpy`, `pandas` and `scikit-learn`. If you're interested specifically in using **_TensorFlow_** or **_PyTorch_**, we also have `jupyterlab-tensorflow` and `jupyterlab-pytorch` which come with those tools pre-installed.
+The jupyter notebooks are great launchpads for analytics including machine learning. The `jupyterlab-cpu` image gives a good core experience for python, including common packages such as `numpy`, `pandas` and `scikit-learn`. If you're interested specifically in using **_TensorFlow_**, we also have `jupyterlab-tensorflow` which come with those tools pre-installed.
 
-For the `jupyterlab-pytorch` image, the PyTorch packages (torch, torchvision, and torchaudio) are installed in the `torch` conda environment. You must activate this environment to use PyTorch.
+For users interested in working with PyTorch, there is support for quickly installing the required packaged.
+in the terminal on your created notebook, run the following command:
+`mamba clean -a -y && mamba create -n torch && mamba install -n torch -c conda-forge -c pytorch -c nvidia -y --file /usr/local/bin/requirements.txt`
+The PyTorch packages (torch, torchvision, and torchaudio) will be installed in the `torch` conda environment.
+You must activate this environment to use PyTorch.
```

Comment:
Maybe give the command on how to activate the env. Maybe a bit too hand-holding but why not avoid potential questions.